config_templates/gretel/synthetics/synthetic-evaluate.yml

```diff
@@ -0,0 +1,29 @@
+# Template for generating synthetic data, then evaluating it on classification or regression models
```

Comment:
Minor: since we're standardizing around the "blueprint" term, should this be "Blueprint configuration for generating..." instead?

Reply:
sgtm, changed